api/src/Tests/Base/CiabTestCase.php

```diff
@@ -291,6 +296,9 @@ protected function runRequest(
             $serverParams = implode('&', $params);
         }
         $request = $this->createRequest($method, $uri, $serverParams, $token);
+        if ($yamlUri) {
+            $this->validator->validate($request, $yamlUri, $method);
```

Comment:
What happens if you simply passed in the `$uri` here rather than `$yamlUri`? Does it produce an error in that case?

Reply:
it does not like things like `/department/2/announcements` and is expecting things like `/department/{id}/announcements`